app/models/user.rb

```diff
@@ -66,7 +66,11 @@ def self.find_by_name(name)
   end
 
   def self.notifiable_owners
-    where(ownerships: { notifier: true })
+    where(ownerships: { push_notifier: true })
+  end
+
+  def self.confirmed_owners
+    where.not(ownerships: { confirmed_at: nil })
```

Comment:
RFC has a separate column for confirmation status, why are we using the timestamp as a proxy for confirmation?
https://github.com/rubygems/rfcs/pull/25/files#diff-a46036696d0714e7cf9d942bcd640d1aR91